devito/passes/iet/definitions.py

```diff
@@ -214,6 +216,38 @@ def _alloc_mapped_array_on_high_bw_mem(self, site, obj, storage, *args):
 
         storage.update(obj, site, allocs=alloc, frees=free, efuncs=(efunc0, efunc1))
 
+    def _alloc_bundle_struct_on_high_bw_mem(self, site, obj, storage):
```

Comment:
Nitpick: some comments would help reduce the bus factor of this code

Reply:
I agree, if it turns green I will add it to the to-be-rebased #2500